packages/dcos-integration-test/extra/test_units.py

```diff
@@ -0,0 +1,34 @@
+import subprocess
+
+import pytest
+
+
+def test_verify_units():
+    """Test that all systemd units are valid."""
```

Comment:
Yes - this would have catched the limitNOFILE= issue. I confirmed on a centos 7.2 system.